apps/sms/index.html

```diff
@@ -51,8 +50,10 @@
     <script defer src="desktop-mock/async_storage.js"></script>
     <!-- Web Components -->
     <script defer src="/shared/elements/config.js"></script>
+    <script defer src="/shared/js/component_utils.js"></script>
     <link rel="stylesheet" type="text/css" href="app://theme.gaiamobile.org/shared/elements/gaia-theme/gaia-theme.css" />
     <script async src="/shared/elements/gaia-header/dist/gaia-header.js"></script>
+    <script defer src="/shared/elements/gaia_checkbox/script.js"></script>
```

Comment:
We will likely get rid of component_utils in the future, or have the component pull in just what it needs.